src/fe_cell_vector_values.jl

```diff
@@ -0,0 +1,56 @@
+immutable FECellVectorValues{dim, T <: Real, FS <: FunctionSpace, GS <: FunctionSpace, shape <: AbstractRefShape, M} <: AbstractFECellScalarValues{dim, T, FS, GS}
```

Comment:
SHouldnt this subtype `AbstractFECellValues`? weird that the vector values subtypes the scalar values...